pkg/gatherers/clusterconfig/config_maps.go

```diff
@@ -55,10 +55,6 @@ func (g *Gatherer) GatherConfigMaps(ctx context.Context) ([]record.Record, []err
 	records = append(records, monitoringRec...)
 	errs = append(errs, monitoringErrs...)
 
-	clusterConfigV1Rec, clusterConfigV1Errs := gatherClusterConfigV1(ctx, coreClient)
-	records = append(records, clusterConfigV1Rec...)
-	errs = append(errs, clusterConfigV1Errs...)
```

Comment:
I believe we don't want to do this. The task was about fixing the archive path (to be aligned with other config maps) and the corresponding sample archive sample AFAIK.

Reply:
Task notes:
"Since the ConfigMap from the kube-system namespace is not used in any rule at the moment, I actually prefer to change the way it is stored to match the documented structure.

However, I don't know why the kube-system ConfigMap was added in the first place and who might be using it. It might have been added because of a rule."

I can update the structure instead, I'm not sure which one is better.